tasks/populate-release-notes-images/populate-release-notes-images.yaml

```diff
@@ -76,6 +77,21 @@ spec:
                 containerImage="${deliveryRepo}@${digest}"
                 # purl should be pkg:oci/bar@sha256%3Aabcde?arch=amd64&repository_url=registry.redhat.io/foo
                 purl="pkg:oci/${deliveryRepo##*/}@${digest/:/%3A}?arch=${arch}&repository_url=${deliveryRepo%/*}"
+                uniqueTagRegex="(rhel-)?v?[0-9]+\.[0-9]+(\.[0-9]+)?-[0-9]{8,}"
```

Comment:
There is no need to define this again for each iteration of the loop, right? So this can be moved outside of the loop above.